fiftyone/core/utils.py

```diff
@@ -1992,3 +1995,26 @@ def validate_hex_color(value):
 
 
 fos = lazy_import("fiftyone.core.storage")
+
+
+def load_dataset(identifier):
+    """Loads the dataset from the database with the given name or id."""
+    import fiftyone.core.odm as foo
+    import fiftyone.core.dataset as fod
+
+    db = foo.get_db_conn()
+    try:
```

Comment:
querying using id or name, but not both because it's possible after the dataset is renamed, the previous name is used for a different dataset